packages/react-vapor/src/components/limit/Limit.tsx

```diff
@@ -0,0 +1,142 @@
+import classNames from 'classnames';
+import * as React from 'react';
+import {Input} from 'react-vapor';
+
+export interface ILimitOwnProps {
+    title: string;
+    limitLabel?: string;
+    usageNumber?: number;
+    limitNumber?: number;
+    isLimitModifiable?: boolean;
+    isLimitTheGoalToReach?: boolean;
+    isHistoryIncluded?: boolean;
+}
+
+export interface ILimitDispatchProps {
+    onRender?: () => void;
+    onDestroy?: () => void;
+}
+
+export interface ILimitProps extends ILimitOwnProps, ILimitDispatchProps {}
```

Comment:
Interfaces shouldn't be prefixed with `I` - see [docs](https://github.com/coveord/tsjs#interface-no-i)